README.md

```diff
@@ -77,6 +79,31 @@ foreach ($results as $item) {
 }
 ```
 
+### Authentication with Service Accounts ###
+
+> An example of this in action can be seen in [`examples/service-account.php`](examples/service-account.php).
+
+1. Follow the instructions to [Create a Service Account](https://developers.google.com/api-client-library/php/auth/service-accounts#creatinganaccount)
+1. Download the JSON credentials
+1. Set the path to these credentials using the `GOOGLE_APPLICATION_CREDENTIALS` environment variable:
+
+    ```php
+    putenv('GOOGLE_APPLICATION_CREDENTIALS=/path/to/service-account.json');
```

Comment:
Nit: Embedding the environment variable into the code will reduce the benefit from Application Default Credentials.
I think there is ADC support, right?